src/core/include/openvino/core/type.hpp

```diff
@@ -118,9 +118,25 @@ struct AsTypePtr<std::shared_ptr<In>> {
 };
 }  // namespace util
 
+
+namespace frontend {
+namespace jax { class ConversionExtension; }
+namespace onnx { class ConversionExtension; }
+namespace tensorflow { class ConversionExtension; }
+namespace tensorflow_lite { class ConversionExtension; }
+namespace pytorch { class ConversionExtension; }
+namespace paddle { class ConversionExtension; }
+} // frontend 
```

Comment:
@praasz @t-jankowski do you know how to use SFINAE here to avoid having forward declarations in current file?

I wanted to achieve that only files with FE-specific `ConversionExtension` are changed.

Reply:
We can make it with check that they are all inherited from one base:
typename = std::enable_if_t<!std::is_base_of_v<ov::frontend::ConversionExtensionBase, T>>

It does not require forward declaration here

Reply:
changed, thank you!